comp/game/tinker-time.yaml

```diff
@@ -0,0 +1,25 @@
+summary: Organise how arena testing slots will be booked
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+area-owner: game
+
+description: >-
+  There is limited availablity for teams to test their robots in the arenas.
+  We need to plan how we are going to ensure that this time is fairly distributed between teams.
+
+  Tinker time slots in the main arena need to be long enough for teams to properly run through their strategies in the actual arena.
+  Conversely, test arena slots need to be regular enough that teams aren't waiting significant times to be able to test.
+
```

Comment:
This new line hasn't come through, which makes the render look odd:

![image](https://github.com/srobo/recurring-tasks/assets/6527489/fc34642a-3ea5-470a-9060-fce73c0ca3c5)

Reply:
This turned out to be the newlines being stripped when the yaml is loaded. Where blank lines are wanted between paragraphs we have to have two blank lines in the yaml.